packages/app/src/client/components/withSession/pages/AccessDenied.tsx

```diff
@@ -33,6 +34,8 @@ export const AccessDeniedPage = ({ data }: { data: {} }) => {
     ? `Contact your administrator or log out and try again with a different account.`
     : "Contact your administrator or try again with a different account.";
 
+  const isPlaywright = Cookies.get("opstrace-playwright");
```

Comment:
Let's sync about this before we merge. Why do we need to use a cookie? 

The bit I'm missing is why we can't just send the error we get from Auth0 (or any other server-side error) back to the UI and then use the appropriate Error component to show the error. In the case of a 403, we use the AccessDenied component, but for all other errors we could just use our ErrorView component